llvm/tools/llvm-profgen/ProfileGenerator.cpp

```diff
@@ -190,9 +195,7 @@ void ProfileGeneratorBase::showDensitySuggestion(double Density) {
 
   if (ShowDensity)
     outs() << "Minimum profile density for hot functions with top "
```

Comment:
With the new heuristic, this is message is no longer accurate? it's no longer hot functions with top x% total samples.